public/html/shiloh.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE HTML>
+<html>
+	
+
+	<head>
+		<title>Shiloh's Assignment 2 Webpage!</title>
+
+	</head>
+
+	<body>
+		<section id="regulations">
+			<h1>Covid 19 Regulations</h1>
+			<div id="nav">
+				<ul>
+					<li><a href="#hotels">Hotels</a></li>
+					<li><a href="#restaurants">Restaurants</a></li>
+					<li><a href="#contactus">Contact Us</a></li>
+				</ul>
+			</div>
+		</section>
+		<section id="hotels">
+			<h2>Hotels</h2>
+
+			<p>All visitors are subject to a 14 day mandatory quarantine. No transient vaction rental properties may be used as a place or quarantine.</p>
+
+			<a href="https://hawaiicovid19.com/">More Info</a>
+		</section>
+
+		<section id="restaurants">
+			<h2>Restaurants</h2>
+
+			<p>Only 25% capacity or 20 patrons (whichever is more) may be served at restaurants. Masks are required at all times except during meals. Any restaurants that are discovered to be inviolation of these regulations will be subject to fines and possible criminal prosecution!</p>
+			<a href="https://hawaiicovid19.com/">
+			<img 
+			src="https://news.llu.edu/sites/news.llu.edu/files/styles/featured_image_755x425/public/screen_shot_2020-08-11_at_9.15.07_am.png?itok=4wgL0Fl1&c=d840cfde9df530f5582918792e0040a0"
+			alt="A picture of acceptable masks"
+			/>
+			</a>
+		</section>
+		<section id="contactus">
+			<h2>Contact Us</h2>
+			<p>For any question regarding the Covid 19 Policies and Procedures, please submit your inquiry using the form below. Mahalo!</p>
+			<div id="contact_form">
+				<form action="/form.html" method="get">
+					<div>
+						<label for="name">Name</label>
+						<input type="text" name="name" id="name"/>
+					</div>
+					<div>
+						<label for="email">Email</label>
+						<input type="email" name="email" id="email"/>
```

Comment:
lets add a single space `id="email" />`

Reply:
Is it good practice to do this on all self-closing tags?

Reply:
yup!